museum/sampleRecords/itemData.destination.json

```diff
@@ -0,0 +1,362 @@
+      "classificationHierarchy":"Root node for Classification;ILLUSTRATIONS;Etching;building"
+    },
+    {
+      "object_id":"4",
+      "id":"4",
+      "idno":"A1964.26b",
+      "display_label":"BROKEN WALLS, ETCHING",
+      "ca_objects.Description":"Brown wood with gilt line frame; glazed, engraving\/etching of old wooden buildings with Perth Town Hall clock tower in background; seen though tall brick doorway. Signed LR.",
+      "ca_objects.MakersMarks":"van Raalte, Henri Benedictus (HenrI)",
+      "ca_objects.ItemDates":"1917-1919",
+      "ca_objects.EarliestYear":"1917",
+      "ca_objects.LatestYear":"1919",
+      "ca_objects.HistoricalDetails":"Henri Van Raalte b.1881-1929",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Etching on paper",
+      "ca_objects.Dimensions.DimensionsValue":"H: 35.5cm W: 13.7cm. Frame:",
+      "classificationHierarchy":"Root node for Classification;ILLUSTRATIONS;Etching;building"
+    },
+    {
+      "object_id":"5",
+      "id":"5",
+      "idno":"P1999\/1006",
+      "display_label":"COLONIAL HOSPITAL, ST MARY'S CATHEDRAL &amp; R.C. BISHOPS HOUSE",
+      "ca_objects.Description":"Colonial Hospital, St Mary's Cathedral &amp; Roman Catholic Bishops House. Victoria Square c1860s\n\nSite of Royal Perth Hospital",
+      "ca_objects.MakersMarks":"Unknown",
+      "ca_objects.ItemDates":"c1863",
+      "ca_objects.EarliestYear":"1863",
+      "ca_objects.LatestYear":"1863",
+      "ca_objects.SignificanceCriteria":"Historic;Social or Spiritual",
+      "ca_objects.StatementOfSignificance":"Historic: \nSocial:",
+      "ca_objects.Dimensions.DimensionsValue":"Large size 28x22 cm",
+      "classificationHierarchy":"Building;hospitals"
+    },
+    {
+      "object_id":"6",
+      "id":"6",
+      "idno":"P1999.1008copied",
+      "display_label":"THE BUNGALOW, SIR JOHN FORREST'S HOME",
+      "ca_objects.Description":"The Bungalow, Hay Street.\nHome of Sir John and Lady Forrest, left to Lady Forrest by her father Howard Hamersley.\nBuilt by Wittenoom family",
+      "ca_objects.MakersMarks":"unknown",
+      "ca_objects.SignificanceCriteria":"Historic;Social or Spiritual",
+      "ca_objects.StatementOfSignificance":"Historic: \nSocial:",
+      "ca_objects.Dimensions.DimensionsValue":"Large size (30x25cm)",
+      "classificationHierarchy":"Building;houses"
+    },
+    {
+      "object_id":"7",
+      "id":"7",
+      "idno":"P1999.1013Copied",
+      "display_label":"DESCRIPTION OF 'PARK BUNGALOW' HOME OF C.Y. O'CONNOR.",
+      "ca_objects.Description":"Description of 'Park Bungalow', home of C.Y. O'Connor\nCOPIED",
+      "ca_objects.MakersMarks":"unknown",
+      "ca_objects.Importance":"State",
+      "ca_objects.StatementOfSignificance":"historical importance",
+      "ca_objects.Dimensions.DimensionsValue":"Imperial size (25x19 cm)",
+      "classificationHierarchy":"PLACE;Building;description"
```

Comment:
(from previous comment)... for example this one has "PLACE" above "Building".  Also the tertiary value "description" seems odd.